src/main/scala/org/scalacheck/ScalaCheckFramework.scala

```diff
@@ -48,7 +49,7 @@ private abstract class ScalaCheckRunner extends Runner {
   }
 
   abstract class BaseTask(override val taskDef: TaskDef) extends Task {
-    val tags: Array[String] = Array()
+    val tags: Array[String | Null] = Array()
```

Comment:
A Scala field overrides some Java array field. We need to use `Array[T | Null]` instead of `Array[T]` for the overriding to work.